Formula/litecli.rb

```diff
@@ -81,7 +81,7 @@ def install
       VALUES
         ('Homebrew');
     EOS
-    system "/usr/bin/sqlite3 test.db < test.sql"
+    system "sqlite3 test.db < test.sql"
 
     require "pty"
 
```

Comment:
Test will need a workaround for `r.read` behavior on Linux. I've usually just done:
```rb
output = ""
begin
  r.each_line { |line| output += line }
rescue Errno::EIO
  # GNU/Linux raises EIO when read is done on closed pty
end
```